### ISSUE TITLE
prisma-proxy-fetch-client

### ISSUE THREAD

**Issue by mkuchak** [jacksloan/prisma-proxy#8]
The version 0.0.8 has a small problem and is not working. The package published on NPM for this release has the `index.js` file, but the package.json is reading the `index.min.js`, so the library is not initialized.

--

[Another issue](https://github.com/jacksloan/prisma-proxy/blob/370268f5ec32775e88052a1f02f1e051dcb2c07c/libs/prisma-proxy-fetch-client/src/index.ts#L37): sometimes Prisma returns undefined and the library throws an error. Maybe it's better to fix it to not throw error with undefined because that's normal during everyday use.

```diff
      try {
         // null is a valid prisma response for some queries
-        return text === null ? text : JSON.parse(text);
+        return text && JSON.parse(text);
       } catch (e) {
```

--

It would be nice to have a fix. By the way, cool project, it's helping me build a template with Cloudflare Workers. Very thanks! :)

**Comment by jacksloan** [comment]
Thanks, @mkuchak, for digging in. Your fork looks good to me if you want to make a PR from that.

**Comment by mkuchak** [comment]
Hey! I redid the PR only with the necessary changes for the correct functioning of the two libraries.

`prisma-proxy-express-server` also has a [little problem](https://github.com/jacksloan/prisma-proxy/blob/main/libs/prisma-proxy-express-server/src/lib/prisma-proxy-express-server.ts#L126) sending the response which should be in JSON.

```diff
     ][prismaAction];
-    prismaFunc(prismaArgs).then((data: any) => res.send(data));
+    prismaFunc(prismaArgs).then((data: any) => res.json(data));
   });
```

As it currently stands some ORM calls like `const totalUsers = await prisma.user.count()` don't work.

With the PR everything should work correctly.

**Comment by jacksloan** [comment]
Great work, looks good to me! Thanks for taking the time to use the library and discover some edge cases I hadn't considered.

**Comment by jacksloan** [comment]
published in 0.0.9
